jobbergate-agent/README.md

```diff
@@ -4,17 +4,16 @@
 
 To install the package from Pypi simply run `pip install jobbergate-agent`.
 
-
 ## Setup parameters
 
 1. Setup dependencies
```

Comment:
need to replace "whenever" with "whatever"

Reply:
Fixed the syntax. Also, changed the text since the projects are managed by Poetry.

Fixed in [6e83a61](https://github.com/omnivector-solutions/jobbergate/pull/398/commits/6e83a61f553318c3c1cc543f9d78e9e6c274db23)